packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -29,7 +29,7 @@ export class MakeCogGithub extends Github {
 
     // skip pull request if not an urban or rural imagery
     if (newTileSet == null) return;
-    await fsa.write(path, JSON.stringify(newTileSet, null, 2));
+    await fsa.write(path, JSON.stringify(newTileSet, null, 2) + '\n');
```

Comment:
It would be best to use prettier to format this.

https://github.com/linz/basemaps-team/pull/515#discussion_r825531902